mesonbuild/compilers/compilers.py

```diff
@@ -64,7 +64,7 @@
     'swift': ('swift',),
     'java': ('java',),
     'cython': ('pyx', ),
-    'nasm': ('asm',),
+    'nasm': ('asm','nasm',),
```

Comment:
Missing a space after the `,`

```suggestion
    'nasm': ('asm', 'nasm',),
```